src/content/ui/animations/index.md

```diff
@@ -186,10 +164,247 @@ or might partially or completely overlap.
+(between 0.0 and 1.0), to the actual animation value. 
+
+The current value of the `Animation` object can be found in the
+`.value` method. The evaluate function also performs some housekeeping,
+such as ensuring that begin and end are returned when the
+animation values are 0.0 and 1.0, respectively.
+
+#### Tween.animate
+
+To use a `Tween` object, call `animate()` on the `Tween`,
+passing in the controller object. For example,
+the following code generates the
+integer values from 0 to 255 over the course of 500 ms.
+
+<?code-excerpt "animation/animate5/lib/main.dart (IntTween)"?>
+```dart
+AnimationController controller = AnimationController(
+    duration: const Duration(milliseconds: 500), vsync: this);
+Animation<int> alpha = IntTween(begin: 0, end: 255).animate(controller);
+```
+
+:::note
+The `animate()` method returns an [`Animation`][],
+not an [`Animatable`][].
+:::
+
+The following example shows a controller, a curve, and a `Tween`:
+
+<?code-excerpt "animation/animate5/lib/main.dart (IntTween-curve)"?>
+```dart
+AnimationController controller = AnimationController(
+    duration: const Duration(milliseconds: 500), vsync: this);
+final Animation<double> curve =
+    CurvedAnimation(parent: controller, curve: Curves.easeOut);
+Animation<int> alpha = IntTween(begin: 0, end: 255).animate(curve);
+```
+
+### Animation notifications
+
+An [`Animation`][] object can have `Listener`s and `StatusListener`s,
+defined with `addListener()` and `addStatusListener()`.
+A `Listener` is called whenever the value of the animation changes.
+The most common behavior of a `Listener` is to call `setState()`
+to cause a rebuild. A `StatusListener` is called when an animation begins,
+ends, moves forward, or moves reverse, as defined by `AnimationStatus`.
+The next section has an example of the `addListener()` method,
+and [Monitoring the progress of the animation][] shows an
+example of `addStatusListener()`.
+
+## Codelabs, tutorials, and articles
+
+The following resources are a good place to start learning
+the Flutter animation framework. Each of these documents
+shows how to write animation code.
+
+* [Implicit animations codelab][]<br>
+  Covers how to use implicit animations
+  using step-by-step instructions and interactive examples.
+
+* [Animations tutorial][]<br>
+  Explains the fundamental classes in the Flutter animation package
+  (controllers, `Animatable`, curves, listeners, builders),
+  as it guides you through a progression of tween animations using
+  different aspects of the animation APIs. This tutorial shows
+  how to create your own custom explicit animations.
+
+* [Zero to One with Flutter, part 1][] and [part 2][]<br>
+  Medium articles showing how to create an animated chart using tweening.
+
+* [Write your first Flutter app on the web][]<br>
+  Codelab demonstrating how to create a form
+  that uses animation to show the user's progress
+  as they fill in the fields.
+
 ## Other resources
```

Comment:
I think we should also direct them to our gaming resources. Those didn't exist when this page was first written.

Reply:
Totally. Added a link to the Casual games toolkit guide.